src/dot.jl

```diff
@@ -30,7 +30,7 @@ function to_dot{G<:AbstractGraph}(graph::G, stream::IO,attrs::AttributeDict=Attr
             write(stream,"$(vertex_index(vtx,graph))$attrs\n")
         end
         for edge in edges(graph)
-            write(stream,"$(vertex_index(source(edge), graph)) $(edge_op(graph)) $(vertex_index(target(edge), graph))\n")
+            write(stream,"$(vertex_index(source(edge), graph)) $(edge_op(graph)) $(vertex_index(target(edge), graph))$(has_edge_attrs ? string(" ", to_dot(attributes(edge, graph))) : "")\n")
```

Comment:
Hi, I'm not sure we should be calling `to_dot` inside the string interpolation here. Could you maybe just explain a little more on why calling `to_dot(attributes(edge, graph)))` inside the string interpolation is the right thing to do? Sorry, but I am not the most knowledgeable on `to_dot`. Although, I would expect a lower level functio that unpacks the graph edge attributes into a string -- is `to_dot` really the best one?